docs/platforms/javascript/common/install/esm-without-import.mdx

```diff
@@ -20,10 +20,12 @@ supported:
 When running your application in ESM mode, you will most likely want to <PlatformLink to="/install/esm">follow the ESM instructions</PlatformLink>. However, if you want to avoid using the `--import` command line option, for example if you have no way of configuring a CLI flag, you can also follow an alternative setup that involves importing the `instrument.mjs` file directly in your application.
```

Comment:
does this link exist? I think the link should point to the --import page, right?

Reply:
Those are the node docs, I just edited those as well as I copied the `Alert` content from there and I thought I add the suggestions I got in this PR in the Node docs as well.

Or should I rather open a new PR for this?